examples/fork/fork.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/io;
+
+public function main() {
```

Comment:
I think we can improve the example. Not entirely clear why one would use fork here over just two workers.

Reply:
+1

Reply:
Was this addressed?

Reply:
Yes, Modified the example to demonstrate fork used with in conditional block.